### PR TITLE
Add SpankBang xPath scraper

### PR DESCRIPTION
**Pull request by halorrr**
Knocking off one of the scrapers from https://github.com/stashapp/CommunityScrapers/issues/73

### CODE REVIEW

File: scrapers/SpankBang.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            - regex: ^
+              with: https://spankbang.com
+          subScraper:
+            selector: //h1
+            postProcess:
+              - replace:
+                - regex: (.+)\sOfficial.+
+                  with: $1
+      Tags:
+        Name: //div[@class='ent']/a[@href[contains(.,'category') or contains(.,'tag')]]/text()|//li[@class='p']/text()
```
Comment (bnkai) [inline scrapers/SpankBang.yml:26]: From the samples i tested it seems that tags always contain the keywords from category and we get a lot of dupes as a result. Was there a case where the `category` was needed ?

File: scrapers/SpankBang.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      Performers:
+        Name: //div[@class='ent']/a[@href[contains(.,'pornstar')]]/text()|//div[@class='left']/h1/a/text()
```
Comment (bnkai) [inline scrapers/SpankBang.yml:28]: The second part of the union seems to add a lot of noise and usually the scenes that don't have the first part do not contain `pro` performers. Should we remove the second part and lose some performers from some scenes but make it more clean ? (`https://spankbang.com/2nma9/video/uncle+i+need+your+help` In some cases like this the performer listed from left class , gina gerson here, isnt even present in the scene).

File: scrapers/SpankBang.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class='left']/h1/@title
+      Details: //meta[@name='description']/@content
+      Image: //div[@class='play_cover']/img/@src
+      Studio:
+        Name:
+          selector: //li[@class='us']/a[@class='ul']/@href[contains(.,'channel')]
```
Comment (bnkai) [inline scrapers/SpankBang.yml:15]: do we need the subscraper ?
it seems to me that 
```yaml
      Studio:
        Name: //li[@class='us']//a[@class='ul'][contains(@href,'channel')]
```
is enough

File: scrapers/SpankBang.yml (hunk 1 above)
Comment (halorrr) [reply scrapers/SpankBang.yml:26]: I didn't notice the repetition on that one, looking at the examples I was referencing that seems to stay true. I'll remove the category

File: scrapers/SpankBang.yml (hunk 2 above)
Comment (halorrr) [reply scrapers/SpankBang.yml:28]: It is really a mixed bag with pornstars when I looked around. There are a ton of scenes with `pro` performers that are only listed in the 2nd area like this one for example `https://spankbang.com/47tsa/video/squirting+down+under` however it also contains performers who aren't in the scene. There doesn't seem to be a good grasp on data accuracy across the site, and at the end of the day, the scraper is as accurate as the site it is scraping from.

Here are two examples where the performers are accurate and only in the 2nd area:
`https://spankbang.com/3glmz/video/how+i+met+my+girlfriend+elsa`
`https://spankbang.com/4f5px/video/pov+cute+blonde+in+2+scenes+swallow+and+creampie`

But to really demonstrate the great (sarcasm) data accuracy, here is one where the pro performers in both sections are entirely wrong:
`https://spankbang.com/32kx6/video/best+friend+bet`

Only so much we can do about that 🤷‍♂️

File: scrapers/SpankBang.yml (hunk 3 above)
Comment (halorrr) [reply scrapers/SpankBang.yml:15]: You are correct! Fixing it

**Pull request merged**